shell.nix

```diff
@@ -2,7 +2,7 @@
 # It just takes the shell attribute from default.nix.
 { config ? {}
 , sourcesOverride ? {}
-, withHoogle ? false
+, withHoogle ? true
```

Comment:
I am not sure what its effect is. Longer build times?
better not change the default.

Reply:
The effect is only on `nix-shell` users -- and it gives people `hoogle` for all the dependencies inside the shell.

It is _tremendously_ useful during development.